apps/sertra/pages/index.tsx

```diff
@@ -3,7 +3,7 @@ import Head from "next/head";
 import { SWRConfig } from "swr";
 import { getServerListings } from "../api/hooks";
 import { ApiURLs } from "../api/urls";
-import styles from "./ServerList.module.css";
+import styles from "../components/ServerList.module.css";
```

Comment:
This seems unrelated to the PR, is it here on purpose?

Reply:
'twas broken, so I fixed it, This could've been a separate pr, but it's so small it seemed reasonable to have it with these changes.